applications/dashboard/views/log/table.php

```diff
@@ -83,7 +83,7 @@
                     ?>
                 </td>
                 <td class="PostType">
-                    <?php echo t($RecordLabel, t('Unknown')); ?>
+                    <?php echo t($RecordLabel, htmlspecialchars($RecordLabel)); ?>
                 </td>
```

Comment:
I thought we were going to remove the t function

Reply:
We still have RecordLabel of 'User', 'Activity' that has translation.

Reply:
The problem was the t function inside the t function.